lib/timer_face/timerSlice.dart

```diff
@@ -0,0 +1,123 @@
+import 'package:flutter/material.dart';
+import '../theme.dart';
+import 'package:fl_chart/fl_chart.dart';
+import '../tasks/TaskList.dart';
+import '../TimeKeeper.dart';
+
+/* 
+timerSlice.dart merely exists to take pressure off of OrgComponents
+by housing getChartSections, which takes the context and timer duration
+and returns a list of <PieChartSectionData> that PieChart uses for 
+each timer slice. 
+*/
+
+int colorValue = 300; //Increment by 100 to change shade
+List<Color> sliceColors = [
+  //Colors for task slices
+  CustomColor.red[colorValue],
+  CustomColor.purple[colorValue],
+  CustomColor.blue[colorValue],
+  CustomColor.green[colorValue],
+  CustomColor.orange[colorValue],
+  CustomColor.pink[colorValue],
+];
+
+// Turns tasks from tasklist into a list of slices, which is used by pie chart
+// Needs context for tasklist and duration to calculate time not used by tasks
+List<PieChartSectionData> getChartSections(
+    BuildContext context, Duration duration) {
+  TaskList taskList = TimeKeeper.of(context).taskList;
+  int listLength = taskList.getLength(); //list length
+  double timeTotal = duration.inSeconds.toDouble();
+  // double timeUsed = 0;
+  Duration timeUsed = Duration(seconds: 0);
+
+  final List<PieChartSectionData> sectionData = [];
```

Comment:
I wouldn't make final if you're planning on adding elements later--signals that you're not planning on changing it. Not sure why Dart allows you to add elements to it later on